From: Ian Jackson <ian.jackson@citrix.com>
Date: Wed, 3 Oct 2018 18:42:42 +0100
Subject: docs/man: Provide properly-formatted NAME sections

A manpage `foo.7.pod' must start with

  =head NAME

  foo - some summary of what foo is or what this manpage is

because otherwise manpage catalogue systems cannot generate a proper
`whatis' entry.

Signed-off-by: Ian Jackson <ian.jackson@citrix.com>
---
 docs/man/xen-pci-device-reservations.pod.7 | 4 ++++
 docs/man/xen-pv-channel.pod.7              | 2 +-
 docs/man/xen-tscmode.pod.7                 | 4 ++++
 docs/man/xen-vtpm.pod.7                    | 6 ++++++
 docs/man/xen-vtpmmgr.pod.7                 | 4 ++++
 docs/man/xl-numa-placement.pod.7           | 2 +-
 6 files changed, 20 insertions(+), 2 deletions(-)

diff --git a/docs/man/xen-pci-device-reservations.pod.7 b/docs/man/xen-pci-device-reservations.pod.7
index dac9276..049e474 100644
--- a/docs/man/xen-pci-device-reservations.pod.7
+++ b/docs/man/xen-pci-device-reservations.pod.7
@@ -1,3 +1,7 @@
+=head1 NAME
+
+xen-pci-device-reservations - Xen PCI device ID registry
+
 =head1 Description
 
 PCI vendor ID 0x5853 has been reserved for use by Xen systems in order to
diff --git a/docs/man/xen-pv-channel.pod.7 b/docs/man/xen-pv-channel.pod.7
index 7229b26..f9f0108 100644
--- a/docs/man/xen-pv-channel.pod.7
+++ b/docs/man/xen-pv-channel.pod.7
@@ -3,7 +3,7 @@
 
 =head1 NAME
 
-Xen PV Channels
+xen-pv-channel - Xen PV Channels
 
 =head1 DESCRIPTION
 
diff --git a/docs/man/xen-tscmode.pod.7 b/docs/man/xen-tscmode.pod.7
index 3bbc96f..819c61d 100644
--- a/docs/man/xen-tscmode.pod.7
+++ b/docs/man/xen-tscmode.pod.7
@@ -1,3 +1,7 @@
+=head1 NAME
+
+xen-tscmode - Xen TSC (time stamp counter) and timekeeping discussion
+
 =head1 OVERVIEW
 
 As of Xen 4.0, a new config option called tsc_mode may be specified
diff --git a/docs/man/xen-vtpm.pod.7 b/docs/man/xen-vtpm.pod.7
index 8de67f4..1d81856 100644
--- a/docs/man/xen-vtpm.pod.7
+++ b/docs/man/xen-vtpm.pod.7
@@ -1,3 +1,9 @@
+=head1 NAME
+
+xen-vtpm - Xen virtual Trusted Platform Module (vTPM) subsystem
+
+=head1 RUBRIC
+
 Copyright (c) 2010-2012 United States Government, as represented by
 the Secretary of Defense.  All rights reserved.
 November 12 2012
diff --git a/docs/man/xen-vtpmmgr.pod.7 b/docs/man/xen-vtpmmgr.pod.7
index 2c3a2de..af825a7 100644
--- a/docs/man/xen-vtpmmgr.pod.7
+++ b/docs/man/xen-vtpmmgr.pod.7
@@ -1,3 +1,7 @@
+=head1 NAME
+
+xen-vtpmgr - Xen virtual TPM stubdomain
+
 =head1 Authors
 
 =over 4
diff --git a/docs/man/xl-numa-placement.pod.7 b/docs/man/xl-numa-placement.pod.7
index 54a4441..ae83302 100644
--- a/docs/man/xl-numa-placement.pod.7
+++ b/docs/man/xl-numa-placement.pod.7
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-Guest Automatic NUMA Placement in libxl and xl
+xl-numa-placement - Guest Automatic NUMA Placement in libxl and xl
 
 =head1 DESCRIPTION
 
